Common/TVSeries.views.widebanner.default.xml

```diff
@@ -498,21 +498,121 @@
+  eq(#TVSeries.Episode.AudioFormat, 'eac3'), 'eac3',
+  eq(#TVSeries.Episode.AudioFormat, 'DTS'), 'dts',
+  and(eq(#TVSeries.Episode.AudioFormat, 'DTS'), eq(#TVSeries.Episode.AudioFormatProfile, 'MA')), 'dtshd_ma',
+  and(eq(#TVSeries.Episode.AudioFormat, 'DTS'), eq(#TVSeries.Episode.AudioFormatProfile, 'HRA')), 'dtshd_hra',
+  eq(string.contains(#TVSeries.Episode.AudioFormatProfile, 'TrueHD'), 'true'), 'truehd',
+  eq(#TVSeries.Episode.AudioFormat, 'MPEG Audio'), 'mp3',
+  eq(string.contains(#TVSeries.Episode.AudioFormat, 'FLAC'), 'true'), 'flac',
+  eq(#TVSeries.Episode.AudioFormat, 'AAC'), 'aac',
+  eq(#TVSeries.Episode.AudioFormat, 'VORBIS'), 'vorbis',
+  eq(string.contains(#TVSeries.Episode.AudioFormat, 'REAL'), 'true'), 'RA',
+  eq(#TVSeries.Episode.AudioFormat, 'WMA'), 'wma')).png</texture>
+    <keepaspectratio>yes</keepaspectratio>
+         <visible>facadeview.list+Control.IsVisible(1239)</visible>
+      <animation effect="fade" time="250">WindowOpen</animation>
+      <animation effect="fade" time="250">WindowClose</animation>
+  </control>
+    <control Style="MediaLogos">
+      <description>Audio channels</description>
+    <type>image</type>
+    <id>0</id>
+        <posX>905</posX>
+        <posY>890</posY>
+    <width>92</width>
+    <height>60</height>
+    <texture>Logos\audio\#(switch(
+  eq(#TVSeries.Episode.AudioChannels, '2'), '2.0',
+  eq(#TVSeries.Episode.AudioChannels, '6'), '5.1',
+  eq(#TVSeries.Episode.AudioChannels, '7'), '6.1',
+  eq(#TVSeries.Episode.AudioChannels, '8'), '7.1')).png</texture>
+    <keepaspectratio>yes</keepaspectratio>
+         <visible>facadeview.list+Control.IsVisible(1239)</visible>
+      <animation effect="fade" time="250">WindowOpen</animation>
+      <animation effect="fade" time="250">WindowClose</animation>
+  </control>
+ 
+    <control Style="MediaLogos">
+      <description>Aspect ratio</description>
+    <type>image</type>
+    <id>0</id>
+        <posX>975</posX>
+        <posY>890</posY>
+    <width>92</width>
+    <height>60</height>
+    <texture>Logos\aspectratio\#(iif(gte(#TVSeries.Episode.VideoAspectRatio, '1.5'),'widescreen','fullscreen')).png</texture>
+    <keepaspectratio>yes</keepaspectratio>
+         <visible>facadeview.list+Control.IsVisible(1239)</visible>
+      <animation effect="fade" time="250">WindowOpen</animation>
+      <animation effect="fade" time="250">WindowClose</animation>
+  </control>
+    <control Style="MediaLogos">
+      <description>Subtitle</description>
+      <type>image</type>
+      <id>0</id>
+      <posX>1065</posX>
+      <posY>900</posY>
+      <width>39</width>
+      <height>40</height>
+      <texture>Logos\#(iif(eq(#TVSeries.Episode.AvailableSubtitles, 0),'nosubtitles','subtitles')).png</texture>
+      <visible>facadeview.list+Control.IsVisible(1239)</visible>
+      <animation effect="fade" time="250">WindowOpen</animation>
+      <animation effect="fade" time="250">WindowClose</animation>
+    </control>

```

Comment:
Not sure what you mean by women's logos? The existing method uses image paths which are in the wrong position, wrong size and there are missing entries for existing images. I also found the CDATA hard to read with all those dashes. This method gives control over the position and size.

Reply:
> Not sure what you mean by women's logos?

Hard - logos :)


> The existing method uses image paths which are in the wrong position, wrong size and there are missing entries for existing images.

There is a fixed size, i.e. a set of images defined in CDATA is collected into one single image and displayed in the position in which it is defined in the skin. Because this can be customized individually, for example, I don’t need subtitles, I don’t have them, I just remove them in TVSeries.SkinSettings.xml and that’s it. Otherwise I will have to change the skin file every time.

Reply:
The width of the standard resolution icon in Titan skin is 145 pixels.

Reply:
Do you mean there's a setting to disable the subtitle image in MP-TVSeries? Where is that?

Reply:
> Do you mean there's a setting to disable the subtitle image in MP-TVSeries? Where is that?

TVSeries.SkinSettings.xml - Remove:
```
skin\Titan\Media\Logos\subtitles.png;-;<Episode.AvailableSubtitles>;-;=;-;1;-;OR;-;<Episode.EpisodeFilename>;-;contains;-;subbed.;-;AND;-;;-;=;-;;-;
skin\Titan\Media\Logos\nosubtitles.png;-;<Episode.AvailableSubtitles>;-;=;-;0;-;AND;-;;-;=;-;;-;
```
from `<logos>`

Reply:
Sure, that would do it, not that I understand why you would bother to remove the subtitles logo.
You could also do the same to the other method using the visible false line.

`    <control Style="MediaLogos">
      <description>Subtitle</description>
      <type>image</type>
      <id>0</id>
      <posX>1065</posX>
      <posY>900</posY>
      <width>39</width>
      <height>40</height>
      <texture>Logos\#(iif(eq(#TVSeries.Episode.AvailableSubtitles, 0),'nosubtitles','subtitles')).png</texture>
      <visible>facadeview.list+Control.IsVisible(1239)</visible>
      <animation effect="fade" time="250">WindowOpen</animation>
      <animation effect="fade" time="250">WindowClose</animation>
	  <visible>false</visible>
    </control>`

Reply:
> I don’t need subtitles, I don’t have them, I just remove them in TVSeries.SkinSettings.xml and that’s it. Otherwise I will have to change the skin file every time.

Not sure what you mean, just add a visible false line to subtitles in TVSeries.views.widebanner.default.xml and that's it as well. Why would you need to do it every time?

Reply:
> Why would you need to do it every time

Each Theme update can update this file. Well, adding or removing any logo is easier in the settings than adding a block to xml. Well, this is in my opinion, we don’t do the same in Moving Pictures or in properties like Studio, Awards, Genres, etc.